examples/experimental/torch/classification/bootstrap_nas.py

```diff
@@ -138,9 +168,20 @@ def validate_model_fn(model_, loader):
                                                                          resuming_checkpoint_path)
 
     if 'train' in config.mode:
-        training_algorithm.run(train_epoch_fn, train_loader, lr_scheduler,
-                               validate_model_fn, val_loader, optimizer,
-                               config.checkpoint_save_dir, config.tb)
+        nncf_network, elasticity_ctrl = training_algorithm.run(train_epoch_fn, train_loader,
+                                                               validate_model_fn, val_loader, optimizer,
+                                                               config.checkpoint_save_dir, config.tb,
+                                                               config.train_steps)
+
+        if resuming_checkpoint_path is None:
+            search_algo = SearchAlgorithm.from_config(nncf_network, elasticity_ctrl, nncf_config)
+        else:
+            search_algo = SearchAlgorithm.from_checkpoint(nncf_network, elasticity_ctrl, bn_adapt_args, resuming_checkpoint_path)
+
+        elasticity_ctrl, best_config, performance_metrics = search_algo.run(validate_model_fn_top1, val_loader, config.checkpoint_save_dir, tensorboard_writer=config.tb)
+
+        print(best_config)
+        print(performance_metrics)
```

Comment:
please use nncf_logger with the appropriate indicator of this log

Reply:
@wonjuleee Added logger. Thanks!